Loop/View Models/BolusEntryViewModel.swift

```diff
@@ -104,6 +104,12 @@ final class BolusEntryViewModel: ObservableObject {
         self.potentialCarbEntry = potentialCarbEntry
         self.selectedCarbAbsorptionTimeEmoji = selectedCarbAbsorptionTimeEmoji
 
+        NotificationCenter.default.addObserver(forName: UIApplication.didBecomeActiveNotification, object: nil, queue: nil) { [weak self] _ in
+            DispatchQueue.main.async {
+                self?.updateChartDateInterval()
```

Comment:
Bolus entry is already invalidated by new data, and without new data, the prediction is still valid. I think this is unneeded.

Reply:
Just tested this again to confirm and you are correct. Not sure why I thought I wasn't noticing the update yesterday. 🤷